test/unit/annotation_spec.js

```diff
@@ -90,16 +91,19 @@ describe('annotation', function() {
         startObjId: 0,
       });
 
-      var annotation1 = AnnotationFactory.create(xref, annotationDict,
-                                                 pdfManagerMock, idFactory);
-      var annotation2 = AnnotationFactory.create(xref, annotationDict,
-                                                 pdfManagerMock, idFactory);
-      var data1 = annotation1.data, data2 = annotation2.data;
-      expect(data1.annotationType).toEqual(AnnotationType.LINK);
-      expect(data2.annotationType).toEqual(AnnotationType.LINK);
+      AnnotationFactory.create(xref, annotationDict,
+                pdfManagerMock, idFactory).then((annotation1) => {
+        var data1 = annotation1.data;
```

Comment:
Since it's scoped, there is no need to call this `data1` and `annotation1`. I think you can just use `data` and `annotation` here if you use `let` instead of `var` (the latter applies in your other changes in this file too).

Reply:
Yes, I see.